playground/src/Playground.tsx

```diff
@@ -17,7 +17,7 @@ import {
 import { defaultConfig, parse, debounce } from './common';
 import * as specs from './specs';
 
-const defaultSchema = specs.streetlights;
+const defaultSchema = specs.oneOf;
```

Comment:
you sure?

Reply:
I forget about it, changed to streetlight.